tardis/plugins/elasticsearchmonitoring.py

```diff
@@ -38,16 +40,22 @@ async def notify(self, state: State, resource_attributes: AttributeDict) -> None
         :type resource_attributes: AttributeDict
         :return: None
         """
+
         self.logger.debug(
             f"Drone: {str(resource_attributes)} has changed state to {state}"
         )
 
+        if "resource_status" in resource_attributes:
+            resource_status = resource_attributes["resource_status"]
+        else:
+            resource_status = ""
+
         document = {
             **resource_attributes,
             "state": str(state),
             "meta": self._meta,
             "timestamp": int(time() * 1000),
-            "resource_status": str(resource_attributes["resource_status"]),
+            "resource_status": str(resource_status),
```

Comment:
You could use `resource_attributes.get("resource_status", "")` instead, in order to avoid the `if` `else` clause above.